pulsebot/bugzilla.py

```diff
@@ -17,85 +17,50 @@ class BugzillaError(Exception):
 class Bugzilla(object):
     def __init__(self, server, api_key):
         self._server = server.rstrip("/")
-        self._session = requests.Session()
-        self._session.headers.update({"User-Agent": "pulsebot"})
-        self._session.headers.update({"X-Bugzilla-API-Key": api_key})
-
-    def get_fields(self, bug, fields):
-        bug_url = "%s/rest/bug/%d?include_fields=%s" % (
-            self._server,
-            bug,
-            ",".join(fields),
-        )
-        logger.info(f"get_fields: {bug_url}")
+        self._headers = {
+            "User-Agent": "pulsebot",
+            "X-Bugzilla-API-Key": api_key,
+            "Content-Type": "application/json",
+            "Accept": "application/json"
+        }
+
+    def _call(self, method, path, data={}):
+        if not method:
+            method = "GET"
```

Comment:
```suggestion
    def _call(self, path, method="GET", data=None):
    data = data or {}
```

There's a classic Python gotcha when putting mutable arguments as defaults. See [this](https://docs.python.org/3/tutorial/controlflow.html#default-argument-values) for more info.